mods/sgssb/statuses.js

```diff
@@ -275,4 +275,17 @@ exports.BattleStatuses = {
 			this.add('c', '+Xavier1942', 'Nuuuuu! MY BEAUTIFUL WALL! ');
 		},
 	},
+	eelek: {
+		exists: true,
+		onStart: function () {
+			this.add('c', '+Eelek', 'Oh hey there! :3');
+		},
+		onSwitchOut: function (pokemon) {
+			this.add('c', '+Eelek', 'Gotta go, I\'ll be back when I get to the cemetery.');
```

Comment:
not necessary, but you could use double quotes around the entire sentence and avoid having to escape the `'` character - it's just more readable